pwa/src/context/tabs.ts

```diff
@@ -8,6 +8,7 @@ export interface ITabsContext {
   objectDetailTabs: number;
   userDetailTabs: number;
   organizationDetailTabs: number;
+  mappingDetailTabs: number;
```

Comment:
As discussed in our call: maybe we can add state to also keep track of the accordions within this page, like so (dummy code):

```
mappingDetailTabs: {
  currentTab: number;
  mappingAccordionIsOpen: boolean;
  unsetAccordionIsOpen: boolean;
  castAccordionIsOpen: boolean;
}
```